test/units/modules/storage/netapp/test_na_ontap_gather_facts.py

```diff
@@ -65,20 +66,49 @@ def invoke_successfully(self, xml, enable_tunneling):  # pylint: disable=unused-
         self.xml_in = xml
         print(xml.to_string())
         if self.type == 'vserver':
-            xml = self.build_vserver_info(self.parm1)
+            xml = self.build_vserver_info()
+        elif self.type == 'net_port':
+            xml = self.build_net_port_info()
+        elif self.type == 'zapi_error':
+            error = netapp_utils.zapi.NaApiError('test', 'error')
+            raise error
         self.xml_out = xml
-        print(xml.to_string())
         return xml
 
     @staticmethod
-    def build_vserver_info(vserver):
+    def build_vserver_info():
```

Comment:
Is this code repeated below? specficially lines 89-96?

Reply:
Yes, good catch.   We can delete one of the two functions, they are now identical.

Reply:
👍